packages/components/src/components/domainproperties/list/ListDesignerPanels.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import {ListDesignerPanels} from "./ListDesignerPanels";
+import {mount} from "enzyme";
+import {ListModel} from "./models";
+import {DEFAULT_LIST_SETTINGS} from "../../../test/data/constants";
+import getDomainDetailsJSON from "../../../test/data/property-getDomainDetails.json";
+import DomainForm from "../DomainForm";
+import {ListPropertiesPanel} from "./ListPropertiesPanel";
+import {Alert} from "react-bootstrap";
+import renderer from 'react-test-renderer';
+import toJson from "enzyme-to-json";
+
+const emptyNewModel = ListModel.create(null, DEFAULT_LIST_SETTINGS);
+const populatedExistingModel = ListModel.create(getDomainDetailsJSON);
+
+describe('ListDesignerPanel', () => {
+    test('new list', () => {
+        const listDesignerPanels =
+            <ListDesignerPanels
+                onComplete={jest.fn()}
+                onCancel={jest.fn()}
+                initModel={emptyNewModel}
+            />;
+
+        const tree = renderer.create(listDesignerPanels).toJSON();
+        expect(tree).toMatchSnapshot();
+    });
+
+    test('existing list', () => {
+        const listDesignerPanels = mount(
+            <ListDesignerPanels
+                onComplete={jest.fn()}
+                onCancel={jest.fn()}
+                initModel={populatedExistingModel}
+            />
+        );
+
+        expect(toJson(listDesignerPanels)).toMatchSnapshot();
```

Comment:
Looks like one more test case to change to the `        const tree = renderer.create(listDesignerPanels).toJSON();
        expect(tree).toMatchSnapshot();` syntax